tests/models/tile_model_tests.py

```diff
@@ -45,18 +45,32 @@ class TileTests(ArchesTestCase):
         "rdf_export_document_model",
         "rdf_export_object_model",
         "Cardinality Test Model",
+        "All_Datatypes",
     ]
 
     @classmethod
     def setUpTestData(cls):
         super().setUpTestData()
-        ResourceInstance.objects.create(
-            pk="40000000-0000-0000-0000-000000000000",
-            legacyid="40000000-0000-0000-0000-000000000000",
-            graph_id="2f7f8e40-adbc-11e6-ac7f-14109fd34195",
-            createdtime="1/1/2000",
-            resource_instance_lifecycle_state_id="4e2a6b8e-2489-4377-9c9f-29cfbd3e76c8",
+        resources = []
+        resources.append(
+            ResourceInstance(
+                pk="40000000-0000-0000-0000-000000000000",
+                legacyid="40000000-0000-0000-0000-000000000000",
+                graph_id="2f7f8e40-adbc-11e6-ac7f-14109fd34195",
+                createdtime="1/1/2000",
+                resource_instance_lifecycle_state_id="4e2a6b8e-2489-4377-9c9f-29cfbd3e76c8",
+            )
```

Comment:
Since these are model instances and not filter() calls, can we use this opportunity to make all of these python UUID types?